### DISCUSSION TITLE
Remove built-in themes?

### DISCUSSION THREAD

**Discussion by canadaduane** [zed-industries/zed#12573] [Feature Requests]
What's the easiest way to remove built-in themes? I love exploring dark themes, but as I cycle through the list of themes, every time a bright white theme flashes through, I feel like I've been spotted by a police officer who is shining a flashlight in my eyes. I'd like to remove these from the list so I can explore without being caught.

**Comment by versecafe** [comment]
@canadaduane you'd have to rebuild the app after deleting the unwanted themes from `assets/themes/` once the app is built they get bundled up into the binary inside a file like `assets-08fb46848ab4b9f5.d`

**Reply by canadaduane** [reply]
I see, thanks.
